### PR TITLE
fixed logo

### PR DESCRIPTION
**Pull request by unnati611**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -269,111 +269,111 @@ <h2 class="font-work-sans font-semibold pl-8 text-2xl text-center">
                         class="w-32 flex flex-col justify-center items-center rounded-xl font-work-sans p-2 cursor-pointer"
                     >
                         <img
-                            src="./resources/assets/jsloversLogo.png"
+                            src="./resources/assets/microsoft_logo.png"
```
Comment (Neha) [inline index.html:272]: Please remove the logo from the file name.

File: index.html [hunk 2]
```diff
@@ -269,111 +269,111 @@ <h2 class="font-work-sans font-semibold pl-8 text-2xl text-center">
...
                             alt="jslovers homepage"
```
Comment (Neha) [inline index.html:273]: Please update all alt tags. We can leave the alt tag empty as the text below this is already having the information

File: index.html [hunk 3]
```diff
@@ -269,111 +269,111 @@ <h2 class="font-work-sans font-semibold pl-8 text-2xl text-center">
...
                             width="100"
                         />
 
-                        <h4 class="mt-2 underline">Microsoft</h4>
+                        <!-- <h4 class="mt-2 underline">Microsoft</h4> -->
```
Comment (Neha) [inline index.html:277]: Please remove the commented code

File: index.html [hunk 4]
```diff
@@ -269,111 +269,111 @@ <h2 class="font-work-sans font-semibold pl-8 text-2xl text-center">
...
                     </div>
                     <div
                         class="w-32 flex flex-col justify-center items-center rounded-xl font-work-sans p-2 cursor-pointer"
                     >
                         <img
-                            src="./resources/assets/jsloversLogo.png"
+                            src="./resources/assets/expedia_logo.svg"
```
Comment (Neha) [inline index.html:283]: can we please follow only one image extension - PNG or SVG

File: index.html [hunk 5]
```diff
@@ -269,111 +269,111 @@ <h2 class="font-work-sans font-semibold pl-8 text-2xl text-center">
...
+                        <!-- <h4 class="mt-2 underline">Expedia</h4> -->
                     </div>
                     <div
                         class="w-32 flex flex-col justify-center items-center rounded-xl font-work-sans p-2 cursor-pointer"
                     >
                         <img
-                            src="./resources/assets/jsloversLogo.png"
+                            src="./resources/assets/google_logo.png"
                             alt="jslovers homepage"
                             width="100"
```
Comment (Neha) [inline index.html:296]: Can we please put a class and there we assign the width rather than having inline CSS?

**Review by Neha — COMMENTED** [review]
Left a few comments. Please check

File: index.html (hunk 2 above)
Comment (unnati611) [reply index.html:273]: ok

File: index.html (hunk 1 above)
Comment (unnati611) [reply index.html:272]: done

File: index.html (hunk 3 above)
Comment (unnati611) [reply index.html:277]: done

File: index.html (hunk 5 above)
Comment (unnati611) [reply index.html:296]: done

**Comment by unnati611** [comment]
@Neha please check changes

File: index.html [hunk 6]
```diff
@@ -269,117 +269,101 @@ <h2 class="font-work-sans font-semibold pl-8 text-2xl text-center">
                         class="w-32 flex flex-col justify-center items-center rounded-xl font-work-sans p-2 cursor-pointer"
                     >
                         <img
-                            src="./resources/assets/jsloversLogo.png"
-                            alt="jslovers homepage"
-                            width="100"
+                            src="./resources/assets/microsoft.png"
+                            class="company_logo"
```
Comment (Neha) [inline index.html:273]: We don't need to add custom CSS. We are using tailwind. We can add tailwind classes here

File: index.html [hunk 7]
```diff
@@ -269,117 +269,101 @@ <h2 class="font-work-sans font-semibold pl-8 text-2xl text-center">
...
 
-                        <h4 class="mt-2 underline">Naggaro</h4>
+                     
                     </div>
                     <div
                         class="w-32 flex flex-col justify-center items-center rounded-xl font-work-sans p-2 cursor-pointer"
                     >
                         <img
-                            src="./resources/assets/jsloversLogo.png"
-                            alt="jslovers homepage"
```
Comment (Neha) [inline index.html:328]: do not remove alt tag. Leaving alt tag empty means alt =""

File: index.html [hunk 8]
```diff
@@ -269,117 +269,101 @@ <h2 class="font-work-sans font-semibold pl-8 text-2xl text-center">
...
-                            width="100"
+                            src="./resources/assets/google.png"
+                            class="company_logo"
                         />
 
-                        <h4 class="mt-2 underline">Google</h4>
```
Comment (Neha) [inline index.html:299]: Why we have removed the H4?

**Review by Neha — COMMENTED** [review]
Left a few changes:

1. Please use the tailwind css class names
2. Remove style.css
3. Remove 1280px-Ticketmaster image from the assets folder
4. alt tag should be empty : alt=""
5. We need H4 tag

**Comment by unnati611** [comment]
please check

**Comment by unnati611** [comment]
image

**Pull request merged**